includes/class-relme-widget.php

```diff
@@ -77,4 +79,21 @@ public function form( $instance ) {
 		esc_html_e( 'Displays rel=me links which appear as icons with the logo of the site linked to when possible', 'indieweb' );
 		echo '</p>';
 	}
+
+	/**
+	 * Fetch the first administrator ID.
+	 *
+	 * @return mixed Administrator user ID.
+	 */
+	public function get_default_admin_author() {
+		$users = get_users(
+			array(
+				'role'   => 'administrator',
+				'number' => 1,
+				'fields' => 'ids',
```

Comment:
shouldn't this be `array('ID')` instead of `ids`?

> fields - Which fields to include in the returned array. Default is 'all'. **Pass an array** of wp_users table fields to return an array of stdClass objects with only those fields.

https://wordpress.stackexchange.com/questions/231003/how-to-get-list-of-all-users-and-their-metadata/231057#231057

Reply:
Yes and no all at once. Looking at https://codex.wordpress.org/Class_Reference/WP_User_Query#Return_Fields_Parameter, it accepts either a string or an array. I believe you can specify exactly which fields to return. However, I have it wrong in being `ids` when it should probably be `ID`. I wager I missed this detail by failing to test with multiple users that could be returned. Or it could have just worked anyway, BUT I'm more than willing to edit to match what is documented as expected.